src/crimson/osd/shard_services.cc

```diff
@@ -211,16 +211,16 @@ seastar::future<> ShardServices::send_pg_temp()
     }
     m->pg_temp.emplace(pgid, pg_temp.acting);
   }
-  return seastar::parallel_for_each(std::begin(ms), std::end(ms),
+  auto fut = seastar::parallel_for_each(std::begin(ms), std::end(ms),
```

Comment:
@xxhdx1985126 i created #37910. and i think it's a better fix. could you help review it?

Reply:
@tchaikov I think it's good to go:-)